packages/zosfiles/src/api/methods/list/List.ts

```diff
@@ -161,6 +162,53 @@ export class List {
         }
     }
 
+    /**
+     * Retrieve zfs files
+     *
+     * @param {AbstractSession}  session      - z/OS MF connection info
+     * @param {IZfsOptions} [options={}] - contains the options to be sent
+     *
+     * @returns {Promise<IZosFilesResponse>} A response indicating the outcome of the API
+     *
+     * @throws {ImperativeError} data set name must be set
+     * @throws {Error} When the {@link ZosmfRestClient} throws an error
+     */
+    public static async zfs(session: AbstractSession, options: IZfsOptions = {}): Promise<IZosFilesResponse> {
+        try {
+            let endpoint = posix.join(ZosFilesConstants.RESOURCE,
+                `${ZosFilesConstants.RES_MFS}`);
+            if (options.path) {
```

Comment:
Unless there is a reason to allow both, I would split this into separate API methods for path and filesystem. There could be some confusion if the caller specifies both.